### DISCUSSION TITLE
Time control: more than 24 h

### DISCUSSION THREAD

**Discussion by vovpff** [YetiForceCompany/YetiForceCRM#14460] [How to] [answered]
`$DISALLOW_LONGER_THAN_24_HOURS = false;` config parameter missing in 6.1. How to activate this parameter now?

**Comment by mariuszkrzaczkowski** [comment, marked as answer]
https://gitdeveloper.yetiforce.com/index.php?parent=Settings&module=EventHandler&view=Index
![image](https://user-images.githubusercontent.com/10198654/112460606-63c20000-8d5f-11eb-946d-3101c4332bc7.png)

**Reply by vovpff** [reply]
Thank you!
